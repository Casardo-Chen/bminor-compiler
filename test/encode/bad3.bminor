this is bad as well"