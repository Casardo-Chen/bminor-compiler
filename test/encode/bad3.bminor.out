Input String    : this is bad as well"
Error: empty/not starting with "
Invalid String: this is bad as well"
